nextpipe/uplink.py

```diff
@@ -107,80 +101,58 @@ def __init__(self, client: Client, config: UplinkConfig):
             log_internal("No application ID or run ID found, uplink is inactive.")
         self.client = client
         self._lock = threading.Lock()
-        self.node_state = {}
+        self.flow = {}
         self.changed = False
         self._terminate = False
         self._terminated = False
-        self._pending_node_updates = []
         self._updates_failed = 0
 
-    def post_graph(self, graph: FlowDTO):
+    def _post_node_update(self):
         """
-        Posts the initial graph to the server.
+        Posts node updates to the platform.
         """
-        if self.inactive:
-            return
+        # Get RFC3339 timestamp in UTC
+        timestamp = datetime.datetime.now(datetime.UTC).isoformat()
+        self.flow.updated_at = timestamp
         resp = self.client.request(
-            "POST",
-            f"/v1/applications/{self.config.application_id}/runs/{self.config.run_id}/graph",
-            payload=graph.to_dict(),
+            "PUT",
+            f"/v1/applications/{self.config.application_id}/runs/{self.config.run_id}/flow",
```

Comment:
Endpoint still needs to be aligned.
```suggestion
            f"/v1/internal/applications/{self.config.application_id}/runs/{self.config.run_id}/flow",
```

Reply:
Haha, I just realized myself while testing. :sweat_smile: 
Thanks for confirming again!